about.html

```diff
@@ -0,0 +1,163 @@
+                    competation from 10 June to 7. July</p>
+            </div>
+            <div class="row program_section">
+                <div class="star_logo">
+                    <a class=" nav-link" href="#">Star<small class="active">Educational</small>Society</a>
+                </div>
+            </div>
+        </div>
+    </section>
+    <!-- .......HISTORY SECTION....... -->
+    <section class="history_section py-4 d-flex flex-column justify-content-center align-item-center container">
+        <div class="title d-flex flex-column justify-content-center  align-item-center py-2 px-2">
+            <h6>See the Past of Star Educational Society</h6>
+            <hr class="hr">
+            <p>Take a look to the history of Star Educational Society when Changez Stablished this center</p>
+        </div>
+        <div class="master">
+            <div class=" img_section">
+                <div class="img_cover ">
+                    <div class="layer">
+                        <h1>1999</h1>
+                        <p>Lorem ipsum dolor sit amet consectetur,</p>
+                    </div>
+                </div>
+                <div class="img_cover">
+                    <div class="layer">
+                        <h1>1990</h1>
+                        <p>Lorem ipsum dolor sit amet consectetur,</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section class="container-fluid bg-custom-black">
+        <div class="container py-5 d-md-none">
+            <div class="row">
+                <div class="col text-center text-white">
+                    <h6>Contact</h6>
+                    <hr class="hr">
+                </div>
+            </div>
+            <div class="row text-center">
+                <div class="col text-white m-1"><img src="assets/icons/mozilla.png" width="50" alt="mozilla"></div>
+                <div class="col text-white m-1"><img src="assets/icons/googlekorea.png" width="50" alt="google"></div>
+                <div class="col text-white m-1"><img src="assets/icons/naver.png" width="50" alt="naver"></div>
+                <div class="col text-white m-1"><img src="assets/icons/daumkakaologo.png" width="50"
+                        alt="daumkakaologo"></div>
+                <div class="col text-white m-1"><img src="assets/icons/airbnb.png" width="50" alt="airbnb"></div>
+            </div>
+        </div>
+    </section>
+  <div class="container-fluid footer">
+    <div class="container">
+        <div class="row">
+                <div class="col text-center pt-3">
+                    <a class=" nav-link" href="#">Star<small class="active">Educational</small>Society</a>
+                </div>
+                <div class="col text-center pt-3">
+                    <p class="tx-custom-black py-1 joinUs">Join us for difference</p>
```

Comment:
- [ ] Dear @Elhambasir, Please use a `footer` tag here instead of a `footer class`. This has an advantage when it comes to semantics. Screen readers and SEO needs such information to better understand your page.